BootloaderCommonPkg/Include/Library/ContainerLib.h

```diff
@@ -32,6 +32,8 @@ typedef UINT8 AUTH_TYPE;
 #define CONTAINER_BOOT_SIGNATURE            SIGNATURE_32 ('B', 'O', 'O', 'T')
 #define CONTAINER_MONO_SIGN_SIGNATURE       SIGNATURE_32 ('_', 'S', 'G', '_')
 #define CONTAINER_KEY_HASH_STORE_SIGNATURE  SIGNATURE_32 ('K', 'E', 'Y', 'H')
+#define CONTAINER_IPFW_SIGNATURE            SIGNATURE_32 ('I', 'P', 'F', 'W')

```

Comment:
This should not be in this file.
IPFW is not common.

Reply:
Sure. I would remove this definition.